Demos/DatAgora_PartDieu/ud-viz-context/Dockerfile

```diff
@@ -4,26 +4,43 @@ LABEL maintainer "VCityTeam"
 LABEL source.repo "https://github.com/VCityTeam/UD-Viz-docker/"
 
 RUN apt-get update
-
-######### Install UD-Viz and build it with npm
+ARG GEOSERVER_FQDN
+ENV GEOSERVER_FQDN=${GEOSERVER_FQDN}
+ARG GEOSERVER_PORT
+ENV GEOSERVER_PORT=${GEOSERVER_PORT}
+ARG TILESET_CUSTOM_SERVER
+ENV TILESET_CUSTOM_SERVER=${TILESET_CUSTOM_SERVER}
+ARG UD_VIZ_PORT
+ENV UD_VIZ_PORT=${UD_VIZ_PORT}
+
+######### Install the UD-Viz demo per se (client code) and build it
+# Node version 14 is required. For its installation (on focal) refer to e.g.
+#  https://computingforgeeks.com/install-node-js-14-on-ubuntu-debian-linux/
+RUN apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
 RUN apt-get install -y nodejs
-RUN apt-get install -y npm
 RUN apt-get install -y git
 
-RUN mkdir -p WebDemos
-WORKDIR /WebDemos
-RUN git clone https://github.com/VCityTeam/UD-Viz.git
-WORKDIR /WebDemos/UD-Viz/UD-Viz-Core
-COPY DemoConfigData.json examples/DemoFull/DemoConfigData.json
-COPY index.html ./index.html
-COPY Demo.js examples/DemoFull/Demo.js
-COPY Demo.html examples/DemoFull/Demo.html
-COPY img examples/DemoFull/img
 
-RUN npm install
-RUN npm run build 
+COPY SimpleServer /SimpleServer
+COPY DatAgora_PartDieu /DatAgora_PartDieu
```

Comment:
DatAgora_PartDieu holds two things:
 - a copy of [DemoFull](https://github.com/VCityTeam/UD-Viz-demo/tree/master/DemoFull) that would be commited with this PR 
 - the DatAgora specific modifications of this (copy of) DemoFull, that boils down to two few files : config.json and BaseDemo.js
 
 Remove the copy of DemoFull from this branch, and within the dockerfile
  - clone UD-Viz-demo
  - eventually move (or copy)  DemoFull directory (as opposed to Imuv server) where the logic of the container needs it
  - overwrite config.json and BaseDemo.js
  
The Dockefile would go
RUN git clone https://github.com/VCityTeam/UD-Viz-demo.git (PIN THE VERSION)
RUN cp -r UD-Viz-demo/DemoFull DatAgora_PartDieu
RUN rm -fr UD-Viz-demo
COPY config.json DatAgora_PartDieu/config.json
COPY BaseDemo.json DatAgora_PartDieu/BaseDemo.json